variable/technology/technologies.yaml

```diff
@@ -186,6 +189,74 @@ Capital Cost|Liquids|Oil:
    description: Capital cost of a new oil refining plant
    unit: US$2010/kW
 
+Constant Variable Cost|Electricity|Coal:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Constant Variable Cost|Electricity|Gas:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Constant Variable Cost|Electricity|Nuclear:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Constant Variable Cost|Electricity|Oil:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Energy Charge|Electricity|Energy Storage System:
+   description: Energy charge by ESS for each load level
+   unit:  TWh
+
+Energy Not Serve Cost|Electricity:
+   description: Cost of energy not served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit:  €/MWh
+
+Energy Not Serve|Electricity:
+   description: Energy not served by node
+   unit:  GWh
+
+Fixed Investment Cost|Electricity|Transmission|Line:
```

Comment:
There is already a variable "Investment" (for investment costs), please follow the existing convention.